server/render/index.js

```diff
@@ -104,18 +104,23 @@ export function serverRender( req, res ) {
 		metas = getDocumentHeadMeta( context.store.getState() );
 		links = getDocumentHeadLink( context.store.getState() );
 
-		let reduxSubtrees = [ 'documentHead' ];
+		const cacheableReduxSubtrees = [ 'documentHead' ];
+		let reduxSubtrees;
 
 		// Send redux state only in logged-out scenario
```

Comment:
The comment is no longer true with that code,
whether we're logged in or logged out is not checked.

I'm not sure we need to remove `! context.user`, what if `ui` subtree contains some user specific state? Like the state of a credit card input field?